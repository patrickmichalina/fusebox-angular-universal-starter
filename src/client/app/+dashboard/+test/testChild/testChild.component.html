<mat-card>
    Test Child Project
</mat-card>