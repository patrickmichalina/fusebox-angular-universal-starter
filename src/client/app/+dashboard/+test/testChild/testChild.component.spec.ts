import { async, ComponentFixture, TestBed } from '@angular/core/testing'
import { Component } from '@angular/core'
import { TestChildComponent } from './testChild.component'
import { TestModule } from '../test.module'

describe(TestChildComponent.name, () => {
  let fixture: ComponentFixture<TestChildComponent>

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [TestModule],
      declarations: [TestComponent]
    }).compileComponents()
  }))

  beforeEach(async(() => {
    fixture = TestBed.createComponent(TestChildComponent)
  }))

  afterEach(async(() => {
    TestBed.resetTestingModule()
  }))

  it('should match snapshot', async(() => {
    expect(fixture).toMatchSnapshot()
  }))

  it('should compile', async(() => {
    expect(fixture.nativeElement).toBeDefined()
  }))
})

@Component({
  selector: 'test-component',
  template: '<pm-test-child-1></pm-test-child-1>'
})
class TestComponent {}
