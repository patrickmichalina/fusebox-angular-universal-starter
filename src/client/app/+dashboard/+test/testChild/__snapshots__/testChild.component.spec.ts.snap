// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestChildComponent should match snapshot 1`] = `
<pm-test-child-1>
  <md-card
    class="mat-card"
  >
    
      Test Child Project
  
  </md-card>
</pm-test-child-1>
`;
